src/components/AssetsTableContent.jsx

```diff
@@ -66,25 +66,30 @@ const AssetsTableContent = (props) => {
                 <AssetTypesContainer />
               </ModalComponent>
             </Table.HeaderCell>
-            <Table.HeaderCell>
-              Sub-category
-              <ModalComponent modalTitle="Add Sub-Category">
-                <AddSubCategoryContainer />
-              </ModalComponent>
-            </Table.HeaderCell>
             <Table.HeaderCell>
               Category
               <ModalComponent modalTitle="Add Asset Category">
                 <CategoryContainer />
               </ModalComponent>
             </Table.HeaderCell>
+            <Table.HeaderCell>
+              Sub-category
+              <ModalComponent modalTitle="Add Sub-Category">
+                <AddSubCategoryContainer />
+              </ModalComponent>
+            </Table.HeaderCell>
           </Table.Row>
         </Table.Header>
 
         <Table.Body>
           {
             props.activePageAssets.map((asset) => {
               const assetViewUrl = `assets/${asset.serial_number}/view`;
+
+              asset.asset_code = asset.asset_code ? asset.asset_code : '-';
```

Comment:
I prefer if you don't mutate the object directly, that is, create another object to update the necessary properties. Also,  the logic can be simplified to `asset.asset_code = asset.asset_code ||  '-';`. Therefore, change this to:
```
const updatedAsset = {
  ...asset,
  asset_code: asset.asset_code || '-',
  serial_number: asset.serial_number : '-',
  model_number: asset.model_number|| '-'
}
```

then the return will be:
```
return (
  <TableRowComponent
    {...props}
    ...
    ...
    data={updatedAsset}
    ...
  />

Reply:
Great. Effecting the changes right away